README.md

```diff
@@ -14,3 +14,59 @@ npm run dev
 ```
 
 Once the local server is running, a live-reloading preview of the spec can be viewed at http://localhost:3000.
+
+## Tools and frameworks used in this repo
+* [Dokz](https://dokz.site/docs)
+* [MDX](https://mdxjs.com/)
+* [Chakra-UI](https://chakra-ui.com/docs/getting-started)
+
+# Contributing a specification
+
+### Formatting, names, etc.
+1. The initial spec status is _Draft_.  Please see [Process](#Process) for details.
+1. Choose a target version number of the platform for the spec.
+1. Use the official outline for specifications1. Respect and observe good file organization practices. For example, if your spec relates to DSNP please put the spec in the _DSNP_ folder and not anywhere else.
+1. Spec files are written in markdown (.md) format, and are located in the _pages_ directory.
+1. Use dashes (`-`) between words in file names so that they appear in the Table of Contents correctly.  Use MDX format (`.mdx` extension) if you want to import React components in the page.
+1. When documenting an API, et, use the following field template:
+
+   | name | description | type | required?|
+   | ---|---|---|---|
+   | thisThing | a description of thisThing | bytes | YES |
```

Comment:
Be nice to get consistent on our spacing, but that's more of a linting question which is already in progress.